release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -808,3 +827,20 @@ must also link to the build config object.
 the service catalog. The `controller-manager` and `apiserver` Pods show a status
 of `CrashLoopBackOff`.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821589[*BZ#1821589*])
+
+* Installing {rh-openstack} 16 with Kuryr does not work due to the
+`ovn_controller` crashing after initial setup.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1812009[*BZ#1812009*])
+
+* The Red Hat Virtualization (RHV) machine `instance-state` annotation and the
+`providerStatus.instanceState` status do not always match. This causes the
+client to fail or incorrectly patch the RHV machine status.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1815394[*BZ#1815394*])
+
+* When scaling up a MachineSet on RHV, the new machine gets stuck in the
```

Comment:
"New machines get stuck [better term for this?] in the Provisioned [capitalization necessary?] phase while scaling up a MachineSet on RHV."

I think that the second sentence is superfluous depending on how you replace "get stuck."

Reply:
`Provisioned` is the specific name for that phase, so it probably should be capitalized. I'll surround it with backticks since it's a system term.